ts/screens/wallet/payment/__tests__/ConfirmPaymentMethodScreen.test.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+// import { fireEvent } from "@testing-library/react-native";
+// import "@testing-library/jest-native/extend-expect";
```

Comment:
We should avoid commented code / import. Could you remove all the commented code?

Reply:
Sure I will remove it. Thank you for pointing out